go/iot/arduino/protocol.txt

```diff
@@ -0,0 +1,58 @@
+Protocol
+
+For raspberry Pi, it would firstly register the views and exporters. Only after the initial setup that Pi could respond
+to the record request from the arduino.
+
+For Arduino, after it collects sensor data, it encapsulates the sensor data based on the required Json format and sends
+record request to the Pi. Afterwards it could not send another record request until it receives a positive response from
```

Comment:
How would we deal with the recorded values while blocked waiting for the response from Pi? Do we buffer them or drop them? It would be nice to add some explanation here.